test/csuite/random_directio/util.c

```diff
@@ -44,15 +44,15 @@
  *     Copy a directory, using direct IO if indicated.
  */
 void
-copy_directory(const char *fromdir, const char *todir, bool directio)
+copy_directory(const char *fromdir, const char *todir, bool directio, bool *fatal)
```

Comment:
Perhaps make `copy_directory` return `bool` rather than `void`?

Reply:
Sure. I don't feel strongly either way but it doesn't pollute the arg list.